topmenubar/src/main/java/de/interseroh/tmb/client/TopMenuBarWebApp.java

```diff
@@ -185,13 +184,23 @@ private Widget createCollapseButton(String dataTarget) {
 	 *
 	 * @param headlineText text for adding
 	 */
-	private NavbarBrand createBadge(String headlineText) {
-		NavbarBrand brand = new NavbarBrand();
-		brand.setBadgePosition(BadgePosition.RIGHT);
-		brand.setBadgeText(headlineText);
+//	private NavbarBrand createBadge(String headlineText) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)